connector/connector.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+var app = require('../server/server.js');
+var connector = app.dataSources.db.connector;
+var WorkspaceManager = require('../component/workspace-manager.js');
+
+/**
+ * @class Connector
+ *
+ * performs CRUD operations on the Workspace graph.
+ */
+connector.createModel = function(id, data, cb) {
+  var workspace = WorkspaceManager.getWorkspace();
```

Comment:
How does this work...? I'd expect it to take a workspace id or instance of something...